machine/lenovo/lenovo_g8272/kernel/platform-lenovo-g8272.patch

```diff
@@ -0,0 +1,764 @@
+diff --git a/arch/powerpc/boot/dts/lenovo_g8272.dts b/arch/powerpc/boot/dts/lenovo_g8272.dts
+new file mode 100644
+index 0000000..57c9a64
+--- /dev/null
++++ b/arch/powerpc/boot/dts/lenovo_g8272.dts
+@@ -0,0 +1,571 @@
++/*
++ * Lenovo G8272 Device Tree Source
++ *
++ */
++/dts-v1/;
++
++/ {
++	model = "Lenovo,G8272";
++	compatible = "Lenovo,g8272";
++	#address-cells = <2>;
++	#size-cells = <2>;
++
++	aliases {
++		ethernet0 = &enet1;
++		serial0 = &serial0;
++		serial1 = &serial1;
++		pci0 = &pci0;
++	};
++
++	cpus {
++		#address-cells = <1>;
++		#size-cells = <0>;
++
++		PowerPC,P2020@0 {
++			device_type = "cpu";
++			reg = <0x0>;
++			next-level-cache = <&L2>;
++		};
++		PowerPC,P2020@1 {
++			device_type = "cpu";
++			reg = <0x1>;
++			next-level-cache = <&L2>;
++		};
++	};
++
++	memory {
++		device_type = "memory";
++	};
++
++	mass_storage {
++		device = "mmcblk0";
++	};
++
++	localbus@fffe05000 {
++		#address-cells = <2>;
++		#size-cells = <1>;
++		compatible = "fsl,p2020-elbc", "fsl,elbc", "simple-bus";
++		reg = <0xf 0xffe05000 0 0x1000>;
++		interrupts = <19 2>;
++		interrupt-parent = <&mpic>;
++
++		/* NOR and NAND Flashes */
++		ranges = <0x0 0x0 0xf 0xef000000 0x01000000
++				0x1 0x0 0xf 0xff800000 0x00040000>;
++
++		nor@0,0 {
++			#address-cells = <1>;
++			#size-cells = <1>;
++			compatible = "cfi-flash";
++			reg = <0x0 0x0 0x01000000>;
++			bank-width = <2>;
++			device-width = <1>;
++                       device-tree@0 {
++                           label = "device-tree";
++                           reg = <0x0 0x20000>;
++                       };
++                       u-boot-env@20000 {
++                           label = "uboot-env";
++                           reg = <0x20000 0x20000>;
++                           env_size = <0x2000>;
++                       };
++                       linux-kernel@40000 {
++                           label = "linux-kernel";
++                           reg = <0x40000 0x300000>;
++                       };
++                       rfs@340000 {
++                           label = "root filesystem";
++                           reg = <0x340000 0x500000>;
++                       };
++                       onie@ba0000 {
++                           label ="onie";
++                           reg = <0xba0000 0x3e0000>;
++                       };
++                       u-boot@f80000 {
++                           label = "uboot";
++                           reg = <0xf80000 0x80000>;
++                       };
++                       open@40000 {
++                           label = "open";
++                           reg = <0x40000 0x800000>;
```

Comment:
Exactly. If you install, for example. the demoOS it trashes our (legacy) OS kernel/RFS but that's OK because it would be reinstalled if you ran the Lenovo NOS installer again.